sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -275,15 +275,14 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
         if (!this.isOpen()) {
           await Promise.race([
             this._init(signal),
-            delay(retryTimeoutInMs, { abortSignal })
```

Comment:
If the customer aborts on their signal, why wouldn't we abort the delay as well?

Also, I think we should pass `signal` and not `abortSignal` here as well because we want to abort the delay promise once the `init` one settles.

Reply:
Right we want to abort the delay too.  The problem (and existing test failing) is that if we pass in `signal` or `abortSignal`, end users may get the error "The delay was aborted", which is internal and confusing as they are aborting the operation.  But I think we could pass `aborter.signal` as we call `aborter.abort()` in finally block.

Reply:
I see. A couple of suggestions: I think we can either catch the error from `delay` and rethrow a user-friendly one: `delay(timeout).catch(() => throw nice error).then(() => throw timeout error)`. Or just use `createAbortablePromise` to re-create `delay` with a user-friendly experience. I did the latter in `waitForEvents`.

Reply:
I moved `.finally()` to be on `Promise.race()` block. `aboter.abort()` there should abort `delay()` when `init()` resolves.  If `delay()` timed out it would be fined too.

Reply:
Discussed with @deyaaeldeen offline.  End users's abortSignal is not passed to `delay()` any more.  `delay()` will be aborted after the management operation resolves or rejects.